engine/src/flutter/lib/web_ui/lib/src/engine/semantics/route.dart

```diff
@@ -6,17 +6,8 @@ import '../dom.dart';
 import '../semantics.dart';
 import '../util.dart';
 
-/// Denotes that all descendant nodes are inside a route.
-///
-/// Routes can include dialogs, pop-up menus, sub-screens, and more.
-///
-/// See also:
-///
-///   * [RouteName], which provides a description for this route in the absense
-///     of an explicit route label set on the route itself.
-class SemanticRoute extends SemanticRole {
-  SemanticRoute(SemanticsObject semanticsObject)
-    : super.blank(EngineSemanticsRole.route, semanticsObject) {
+class SemanticRoutBase extends SemanticRole {
+  SemanticRoutBase(EngineSemanticsRole kind, SemanticsObject object) : super.blank(kind, object) {
```

Comment:
This class seems to hard-code "dialog" ARIA. Should we use different ARIA roles for different types of routes? E.g. the `SemanticAlertDialog` could use https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alertdialog_role

Reply:
looks like i missed this when i refactored, will fix